templates/_layout.html

```diff
@@ -14,8 +14,36 @@
       <meta property="og:site_name" content="Snapcraft"/>
       <meta property="og:type" content="website"/>
       <meta property="og:description" content="{{ self.description() }}"/>
-      <meta property="og:image" content="https://assets.ubuntu.com/v1/e635d1ef-snapcraft_green-red_hex.png" />
+      <meta property="og:image" content="{% block meta_image %}https://assets.ubuntu.com/v1/e635d1ef-snapcraft_green-red_hex.png{% endblock %}" />
+      <meta property="og:url" content="https://snapcraft.io/{% block snap %}{% endblock %}" />
```

Comment:
Having a block named `snap` in global layout may be problematic. It's hard to say from its name that it's meant to contain URL-friendly snap package name. I think something more explicit like `meta_snap_name` would be better.